### DISCUSSION TITLE
Increasing swap in BTRFS partition

### DISCUSSION THREAD

**Discussion by Disil** [SpiralLinux/SpiralLinux-project#268] [Q&A] [answered]
Can anyone tell me how to increase swap in BTRFS? I use the preconfigured partition, but now I realize that I need more swap space (because my RAM is only 4gb). I tried to use the usual _swapfile_ metode but it won't be used. My current swap is 4gb/3,5 gb.

**Comment by geckolinux** [comment, marked as answer]
https://btrfs.readthedocs.io/en/latest/Swapfile.html
